tests/fixture/test_ordered_actions.py

```diff
@@ -30,7 +28,7 @@ class Object(Base):
     name = Column(String, nullable=False)
     belongs_to = Column(Integer, ForeignKey("stuffs.user.id"))
 
-    owner: List[User] = relationship("User", back_populates="objects")
+    owner = relationship("User", back_populates="objects", uselist=False)  # type: ignore
```

Comment:
hmm why is mypy acting up now?

Reply:
i'm reasonably certain our existing CI acts weird. that type annotation without `Mapped` isn't correct and sqlalchemy/mypy **should** be warning/erroring according to their docs.

And given that Mapped only exists in 1.4+, and we currently suppoert 1.3, and the mypy compat changes (sqlalchemy-stubs vs sqlalchemy2-stubs vs no stubs native in 2.0) between sqlalchemy version, **and** that this is tests, i figure it wasn't important.